layouts/partials/head.html

```diff
@@ -40,7 +40,7 @@
 <meta property="og:description" content="{{ if .IsHome }}{{ $.Site.Params.Subtitle }}{{ else }}{{ .Summary | plainify }}{{ end }}" />
 <meta property="og:url" content="{{ .Permalink }}" />
 <meta property="og:site_name" content="{{ .Title }}" />
-<meta property="og:image" content="{{ if .IsHome }}{{ $.Site.Params.favicon | absURL }}{{else}}{{ .Params.Cover | absURL }}{{ end }}">
+<meta property="og:image" content="{{ if .IsHome }}{{ $.Site.Params.favicon | absURL }}{{else}}{{ isset .Params "cover" | absURL }}{{ end }}">
```

Comment:
```
{{ isset .Params "cover" | absURL }}
```
It always returns `boolean` ;-) Missing `else if` statement.

I think we should do this with fallback, like:

```suggestion
{{ if and (ne .IsHome) (isset .Params "cover") }}
  <meta property="og:image" content="{{ .Param "cover" | absURL }}">
{{ else }}
  {{ if isset $.Site.Params "favicon" }}
    <meta property="og:image" content="{{ $.Site.Params.favicon | absURL }}">
  {{ else }}
    <meta property="og:image" content="{{ printf "img/favicon/%s.png" $.Site.Params.ThemeColor | absURL }}">
  {{ end }}
{{ end }}
```

And I have to say that this is a little bit tricky. I don't work with Hugo on a daily basis anymore and I forgot that `isset` uses `.Params "cover"`, **but** when you want to use it to get regular value, then it has to be `.Param "cover"` (singular) 🧐

Reply:
This is my first "proper" venture in to Hugo!

So basically, we'd be using the cover image for anything that's not home, and for anything that is home we use the favicon with fallback to the default color block? Seems sensible to me 😄 

I wonder if there's a clever way to de-dupe this but still keep it readable... seems crazy to me that we'd need 9 lines just for `og:image`

Reply:
I know it's crazy but this seems to be the most readable solution out there right now ;-D 

We can always do something like that:

```
{{ $shouldUseCover := (not .IsHome) and (isset .Params "cover") }}
{{ $shouldUseUsersFavicon := isset $.Site.Params "favicon" }}
<meta property="og:image" content="{{ cond $shouldUseCover .Params.Cover ( cond $shouldUseUsersFavicon $.Site.Params.favicon (printf "img/favicon/%s.png" $.Site.Params.ThemeColor) ) | absURL }}">
```

¯\_(ツ)_/¯